_data/investing.yml

```diff
@@ -57,6 +57,12 @@ websites:
       software: Yes
       doc: https://www.interactivebrokers.com/en/index.php?f=ibgStrength&p=log
 
+    - name: John Hancock
+      url: https://www.jhpensions.com
```

Comment:
I cannot access this domain and neither can [Is It Down Right Now](http://www.isitdownrightnow.com/jhpensions.com.html).